include/jemalloc/internal/util.h

```diff
@@ -72,6 +72,7 @@
 #  if JEMALLOC_GNUC_PREREQ(4, 6) ||					\
       JEMALLOC_CLANG_HAS_BUILTIN(__builtin_unreachable)
 #	define unreachable() __builtin_unreachable()
+#	define JEMALLOC_HAS_UNREACHABLE
 #  else
 #	define unreachable()
```

Comment:
It does.

On Wed, Mar 23, 2016 at 9:25 PM, Jason Evans <notifications@github.com>
wrote:

> In include/jemalloc/internal/util.h
> <https://github.com/jemalloc/jemalloc/pull/361#discussion_r57230393>:
>
> > @@ -72,6 +72,7 @@
> >  #  if JEMALLOC_GNUC_PREREQ(4, 6) ||					\
> >        JEMALLOC_CLANG_HAS_BUILTIN(__builtin_unreachable)
> >  #	define unreachable() __builtin_unreachable()
> > +#	define JEMALLOC_HAS_UNREACHABLE
> >  #  else
> >  #	define unreachable()
>
> What if we instead define unreachable() as a call to abort() here? Does
> that avoid the compilation warning?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/jemalloc/jemalloc/pull/361/files/b3fd251236f2962e10b5eaacdd6c66b969af9b2b..82320351da6803676f069403364d1a9e55e1dad6#r57230393>
>